From takeuchi_satoru@jp.fujitsu.com  Fri Jan 19 14:29:23 2007
From: Satoru Takeuchi <takeuchi_satoru@jp.fujitsu.com>
Date: Thu, 18 Jan 2007 13:50:05 +0900
Subject: PCI: cleanup MSI code
To: linux-pci <linux-pci@atrey.karlin.mff.cuni.cz>
Cc: Greg KH <gregkh@suse.de>
Message-ID: <87sle96irm.wl%takeuchi_satoru@jp.fujitsu.com>


Cleanup MSI code as follows:

 - fix some types
 - fix strange local variable definition
 - delete unnecessary blank line
 - add comment to #endif which is far from corresponding #ifdef

Signed-off-by: Satoru Takeuchi <takeuchi_satoru@jp.fujitsu.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/msi.c   |    9 +++++----
 include/linux/msi.h |    3 +--
 2 files changed, 6 insertions(+), 6 deletions(-)

--- gregkh-2.6.orig/drivers/pci/msi.c
+++ gregkh-2.6/drivers/pci/msi.c
@@ -49,8 +49,8 @@ static void msi_set_mask_bit(unsigned in
 	switch (entry->msi_attrib.type) {
 	case PCI_CAP_ID_MSI:
 		if (entry->msi_attrib.maskbit) {
-			int		pos;
-			u32		mask_bits;
+			int pos;
+			u32 mask_bits;
 
 			pos = (long)entry->mask_base;
 			pci_read_config_dword(entry->dev, pos, &mask_bits);
@@ -162,6 +162,7 @@ void unmask_msi_irq(unsigned int irq)
 }
 
 static int msi_free_irq(struct pci_dev* dev, int irq);
+
 static int msi_init(void)
 {
 	static int status = -ENOMEM;
@@ -291,7 +292,7 @@ static int msi_lookup_irq(struct pci_dev
 			continue;
 		spin_unlock_irqrestore(&msi_lock, flags);
 		/* This pre-assigned MSI irq for this device
-		   already exits. Override dev->irq with this irq */
+		   already exists. Override dev->irq with this irq */
 		dev->irq = irq;
 		return 0;
 	}
@@ -458,7 +459,7 @@ void pci_restore_msix_state(struct pci_d
 	pci_write_config_word(dev, msi_control_reg(pos), save);
 	enable_msi_mode(dev, pos, PCI_CAP_ID_MSIX);
 }
-#endif
+#endif	/* CONFIG_PM */
 
 /**
  * msi_capability_init - configure device's MSI capability structure
--- gregkh-2.6.orig/include/linux/msi.h
+++ gregkh-2.6/include/linux/msi.h
@@ -7,11 +7,10 @@ struct msi_msg {
 	u32	data;		/* 16 bits of msi message data */
 };
 
-/* Heper functions */
+/* Helper functions */
 extern void mask_msi_irq(unsigned int irq);
 extern void unmask_msi_irq(unsigned int irq);
 extern void read_msi_msg(unsigned int irq, struct msi_msg *msg);
-
 extern void write_msi_msg(unsigned int irq, struct msi_msg *msg);
 
 struct msi_desc {
